app/browser/menu.js

```diff
@@ -622,17 +634,28 @@ const doAction = (state, action) => {
         const frame = frameStateUtil.getFrameByTabId(state, action.tabId)
         if (frame) {
           currentLocation = frame.location
-          setMenuItemChecked(state, locale.translation('bookmarkPage'), isCurrentLocationBookmarked(state))
+          setMenuItemAttribute(state, locale.translation('bookmarkPage'), 'checked', isCurrentLocationBookmarked(state))
+        }
+        break
+      }
+    case appConstants.APP_WINDOW_CLOSED:
+    case appConstants.APP_WINDOW_CREATED:
+      {
+        const windowCount = BrowserWindow.getAllWindows().filter((w) => w.isFocused()).length
```

Comment:
Please don't use the muon `BrowserWindow` API raw. Instead please use the `app/browser/windows.js` API, specifically `getAllRendererWindows` or `getActiveWindow`. The former should be enough as it makes sure the hidden window isn't included.

Reply:
Good call- I was wondering that myself 😄